packages/manager/modules/billing/src/autoRenew/disable/disable.routing.js

```diff
@@ -34,6 +34,7 @@ export default /* @ngInject */ ($stateProvider) => {
             service.setManualRenew();
             return service;
           }),
+          true,
```

Comment:
To be coherent with this [feedback](https://github.com/ovh/manager/pull/20471#discussion_r2540994066), I would change this function to the following:
```suggestion
      updateRenew: ($q, BillingAutoRenew) => (services) =>
        services.map((service) => {
          BillingAutoRenew.updateRenewMode(service.id, SERVICE_RENEW_MODES.MANUAL);
        )
```